fixtures/parallelizer/__init__.py

```diff
@@ -251,7 +251,9 @@ def __init__(self, config):
             request = self.sprout_client.request_check(self.sprout_pool)
             self.appliances = []
             # Push an appliance to the stack to have proper reference for test collection
-            IPAppliance(address=request["appliances"][0]["ip_address"]).push()
+            # FIXME: this is a bad hack based on the need for controll of collection partitioning
```

Comment:
if we pull initial appliance creation out, it should get more nice

collection shouldn't need an appliance